src/aspire/utils/filters.py

```diff
@@ -163,6 +163,26 @@ def _evaluate(self, omega):
         return res
 
 
+class ScaledFilter(Filter):
+    """
+    A Filter object that is composed of a regular `Filter` object, but evaluates it to scale omega.
+    """
+    def __init__(self, filt, scale):
+        self._filter = filt
```

Comment:
make a copy of `filt` inside the constructor, so the caller doesn't have to.

Reply:
Done in commit 6321619.